src/WebJobs.Extensions.DurableTask/LinuxAppServiceLogger.cs

```diff
@@ -22,86 +22,11 @@ internal class LinuxAppServiceLogger
     {
         private const string ConsolePrefix = "MS_DURABLE_FUNCTION_EVENTS_LOGS";
 
-        // The ordered list of regex columns in our legacy logging strategy for linux dedicated.
-        private readonly string[] orderedRegexCol = new string[]
-        {
-            "Account",
-            "ActiveActivities",
-            "ActiveOrchestrators",
-            "Age",
-            "AppName",
-            "ContinuedAsNew",
-            "CreatedTimeFrom",
-            "CreatedTimeTo",
-            "DequeueCount",
-            "Details",
-            "Duration",
-            "ETag",
-            "Episode",
-            "EventCount",
-            "EventName",
-            "EventType",
-            "Exception",
-            "ExceptionMessage",
-            "ExecutionId",
-            "ExtensionVersion",
-            "FromWorkerName",
-            "FunctionName",
-            "FunctionState",
-            "FunctionType",
-            "Input",
-            "InstanceId",
-            "IsCheckpointComplete",
-            "IsExtendedSession",
-            "IsReplay",
-            "LastCheckpointTime",
-            "LatencyMs",
-            "MessageId",
-            "MessagesRead",
-            "MessagesSent",
-            "MessagesUpdated",
-            "NewEventCount",
-            "NewEvents",
-            "NextVisibleTime",
-            "OperationId",
-            "OperationName",
-            "Output*",
-            "PartitionId",
-            "PendingOrchestratorMessages",
-            "PendingOrchestrators",
-            "Reason",
-            "RelatedActivityId",
-            "RequestCount",
-            "RequestId",
-            "RequestingExecutionId",
-            "RequestingInstance",
-            "RequestingInstanceId",
-            "Result",
-            "RuntimeStatus",
-            "SequenceNumber",
-            "SizeInBytes",
-            "SlotName",
-            "StatusCode",
-            "StorageRequests",
-            "Success",
-            "TableEntitiesRead",
-            "TableEntitiesWritten",
-            "TargetExecutionId",
-            "TargetInstanceId",
-            "TaskEventId",
-            "TaskHub",
-            "Token",
-            "TotalEventCount",
-            "Version",
-            "VisibilityTimeoutSeconds",
-            "WorkerName",
-        };
-
         // variable below is internal static for testing and other convenient purposes
         // we need to be able to change the logging path for a windows-based CI
         // the logger being internal static is convenient for flushing it
 #pragma warning disable SA1401 // Fields should be private
-        internal static string LoggingPath = "/var/log/functionsLogs/durableevents.log";
+        internal static string LoggingPath = "/var/log/functionsLogs/durableeventsJSON.log";
```

Comment:
Before we merge, we should just 100% verify that this is indeed the file path listened to by linux app services.

Reply:
Confirmed and e2e tested